src/server/Html.js

```diff
@@ -27,7 +27,12 @@ export default function Html({store, entries, StyleSheetServer, renderProps}) {
     );
   });
   const dehydratedStyles = `window.__APHRODITE__ = ${JSON.stringify(css.renderedClassNames)}`;
-  const auth0ClientOptions = `window.__AUTH0__ = ${JSON.stringify(auth0)}`;
+  const clientOptions = `
+    window.__ACTION__ = {
+      auth0: ${JSON.stringify(auth0)},
+      sentry: ${JSON.stringify(process.env.SENTRY_DSN_PUBLIC)}
```

Comment:
I don't think so. Not without shifting our build process to our server infrastructure. Webpack's `EnvironmentPlugin` will let us set these variables at compile time, but we want them at run time, inherited from the server's environment. I think this is a reasonable way to proceed for now...